dev-infrastructure/scripts/miwi-pg-connect.sh

```diff
@@ -7,7 +7,7 @@ NAMESPACE=$4
 SA_NAME=$5
 
 # prep creds and configs
-PGHOST=$(az postgres flexible-server list --resource-group ${RESOURCEGROUP} --query "[?starts_with(name, '${DB_SERVER_NAME_PREFIX}')].fullyQualifiedDomainName" -o tsv)
+export PGHOST=$(az postgres flexible-server list --resource-group ${RESOURCEGROUP} --query "[?starts_with(name, '${DB_SERVER_NAME_PREFIX}')].fullyQualifiedDomainName" -o tsv)
 AZURE_TENANT_ID=$(az account show -o json | jq .homeTenantId -r)
 AZURE_CLIENT_ID=$(az identity show -g ${RESOURCEGROUP} -n ${MANAGED_IDENTITY_NAME} --query clientId -o tsv)
 SA_TOKEN=$(kubectl create token ${SA_NAME} --namespace=${NAMESPACE} --audience api://AzureADTokenExchange)
```

Comment:
Should we make the expiration of this token short-lived?  I.e. 24 hrs?

Reply:
* the default SA token on AKS expires in 1h, i made it explicit though
* for `az account get-access-token` there is no way to set the expiration - it defaults to 24h